### PR TITLE
Change first param to the path of the sqlite file

### PR DESCRIPTION
**Pull request by anarellano**

### CODE REVIEW

File: server/api.py [hunk 1]
```diff
@@ -49,8 +52,9 @@ def find(name: str):
 
 @app.get("/history_component")
 def history_component():
-    data = fetch_all()
+    data = fetch_all(db_path)
     formatted_data = []
+    print("bbb", data)
```
Comment (evanugarte) [inline server/api.py]:
```suggestion
```

File: server/api.py [hunk 2]
```diff
@@ -14,6 +15,8 @@
     allow_methods=["*"],  # Allows all methods
     allow_headers=["*"],  # Allows all headers
 )
+args = parse_args()
+db_path = args.sqlite_file_path
```
Comment (evanugarte) [inline server/api.py]: lets just use `args.sqlite_file_path` in the below lines instead of creating a variable

File: server/api.py (hunk 1 above)
Comment (anarellano) [reply server/api.py]: done, thanks

File: server/api.py (hunk 2 above)
Comment (anarellano) [reply server/api.py]: done, thanks

**Review by evanugarte — APPROVED** [review]

**Pull request merged**